specification/sql/resource-manager/Microsoft.Sql/preview/2023-02-01-preview/LocationCapabilities.json

```diff
@@ -420,6 +420,40 @@
         }
       }
     },
+    "FreeLimitExhaustionBehaviorCapability": {
+      "description": "Supported free limit exhaustion behavior options",
+      "type": "object",
+      "properties": {
+        "name": {
```

Comment:
>name [](http://example.com/codeflow?start=9&length=4)

Don't use any of the top-level property names within the property bag. Ref: [https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#resource-request-properties-envelope).